scanpy/get/_aggregated.py

```diff
@@ -28,7 +30,6 @@ class Indices(NamedTuple):
     weight_value: pd.Series | Array | None
 
 
-@dataclass
```

Comment:
Stick with this. You can use `default_factory`, `InitVar` and/or `__post_init__` to do whatever you want, no need to go away from data classes.